src/components/EntityInfo/EntityMenu.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <menu id="entity-menu">
```

Comment:
What do you think about either moving `<menu>` back to the parent or creating a wrapper with a slot so the subcomponents can be listed like we were thinking about doing with the `YourCompany` component?